examples/examples/async-fn.rs

```diff
@@ -7,11 +7,11 @@
 //!
 //! You can test this out by running:
 //!
-//!     ncat -l 6142
+//!     nc -l 6142
 //!
 //! And then in another terminal run:
 //!
-//!     cargo +nightly run --example async_fn
+//!     cargo +nightly run --example async-fn
```

Comment:
this no longer requires nightly rust...async/await has been stable for a while now :)

maybe we should remove the `+nightly` while we're here?
```suggestion
//!     cargo run --example async-fn
```